### PR TITLE
Add simple key-value store, and attach to users

### PR DESCRIPTION
**Pull request by stefanor**
Fixes: #147

Simple KeyValue store implementation. Currently the admin view of this is hacky-as-hell. You can't nest inlines (and these models don't really work well in inlines, anyway), so we're probably going to have to do some major customization of the user edit form, to handle these well.

### CODE REVIEW

File: wafer/users/admin.py [hunk 1]
```diff
@@ -5,15 +5,48 @@
...
+    return obj.user.email
+
+
+class UserProfileAdmin(admin.ModelAdmin):
+    model = UserProfile
+    readonly_fields = ('user',)
+    exclude = ('kv',)
+    inlines = (KVPairsInline,)
+    list_select_related = ('user',)
+    list_display = (username, 'display_name', email)
```
Comment (hodgestar) [inline wafer/users/admin.py]: Do `user__username` and `user__email` not work in `list_display`?

Comment (stefanor) [inline wafer/users/admin.py]: Nope.

**Comment by hodgestar** [comment]
:+1: from me, although I'm a bit unsure about why the last commit doesn't have a build.

**Comment by stefanor** [comment]
Because GitHub orders by commit date, not graph order.

**Pull request merged**